src/main/java/tech/allegro/wildsnake/api/ApiRestController.java

```diff
@@ -0,0 +1,51 @@
+package tech.allegro.wildsnake.api;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import tech.allegro.wildsnake.product.model.Product;
+import tech.allegro.wildsnake.product.repository.ProductRepository;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Created by Szymon on 06.12.2015.
+ */
+
+@RestController
+@RequestMapping("/api/v1")
+public class ApiRestController {
+
+    private ProductRepository productRepository;
```

Comment:
It is a good principle to define immutable private fields. So please make the field "final"